### ISSUE TITLE
OpenSubtitlesUpload v2.0

### ISSUE THREAD

**Issue by ivandrofly** [SubtitleEdit/plugins#23]
@niksedk I think we are ready to release `OpenSubtitleUpload` v2 :smiley:

**Comment by niksedk** [comment]
Have you released this?
What's new?

**Comment by ivandrofly** [comment]
> Have you released this?
> What's new?

No, still on branch https://github.com/SubtitleEdit/plugins/tree/OpenSubtitlesUpload can you take a look if you have time?

**Comment by ivandrofly** [comment]
Detecting video frame won't free the UI now
